src/main/java/org/fundacionjala/pivotal/pages/Story.java

```diff
@@ -128,4 +159,56 @@ public boolean existStory(final String text) {
         return false;
     }
 
+    /**
+     * click dropdown story button..
+     *
+     * @param text name of the history
+     */
+    public void clickstoryDropdownButton(final String text) {
```

Comment:
Please move it to Panel page and correct the name to : expandStory()

Reply:
fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0